src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/RedundantImportCheck.java

```diff
@@ -48,6 +48,20 @@
  * &lt;module name="RedundantImport"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * package Test; // OK
```

Comment:
This check do not care of package, remove comment please

Reply:
Done @romani